Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -3192,6 +3192,25 @@ public static int ListEnabledActions(List<InputAction> actions)
         /// </summary>
         /// <value>Current version of the input system.</value>
         public static Version version => new Version(kAssemblyVersion);
+        
+        /// <summary>
+        /// Sets the platform behavior related to running the player in the background or not
+        /// </summary>
+        /// <remarks>
+        /// Some platforms don't care about <see cref="Application.runInBackground"/> and for those we need to
+        /// enable them manually through this method.
+        /// </remarks>
+        /// <param name="value">The boolean value to set to <see cref="NativeInputRuntime.runInBackground"/></param>
+        public static void SetPlatformRunInBackgroundSetting(bool value)
+        {
+                s_Manager.m_Runtime.runInBackground = value;
+        }
+        
+        //TODO remove this, used for debugging only for now
```

Comment:
Remove this if no longer needed. Or make 'internal' as you can setup so tests can get to that.